balboa-common/src/main/java/com/socrata/balboa/metrics/config/Configuration.java

```diff
@@ -45,15 +40,15 @@ public static synchronized Configuration get() throws IOException {
     /**
      * Don't use this, it's only for mocking.
      */
-     public synchronized void setSupportedTypes(List<Period> supportedPeriods) {
+    public synchronized void setSupportedTypes(List<Period> supportedPeriods) {
         this.supportedPeriods = supportedPeriods;
     }
 
     public synchronized List<Period> getSupportedPeriods() {
         if (supportedPeriods == null) {
             String[] types = getProperty("balboa.summaries").split(",");
 
-            supportedPeriods = new ArrayList<Period>(types.length);
+            supportedPeriods = new ArrayList<>(types.length);
```

Comment:
Java 7 syntax diamond syntax. A generic type for ArrayList not required if it can be inferred from the type of the variable. In this case, Java knows `supportedPeriods` has the generic type `Period`, so I don't have to specify the generic type of the `ArrayList`.

Since it's not necessary to specify `Period` with `ArrayList`, Intellij highlights it as a warning, and I like to clear as many warnings as possible. A few times I've run across real errors as I do it.